pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
```

Comment:
We should configure this in the [`pyproject.toml` file that we already have](https://github.com/mautrix/python/blob/master/pyproject.toml). See https://docs.pytest.org/en/6.2.x/customize.html#pyproject-toml for docs on how to configure it properly.

Reply:
I will make the change